Tutorials/NetKet_Tools/plot_sigmax.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018 The Simons Foundation, Inc. - All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Demonstrating the plotting helper functions in NetKet Tools.
+"""
+
+import netket_tools as nkt
+nkt.python_utils.plot_observable("test", "SigmaX", exact=0.637275 * 20)
```

Comment:
This is also very nice! 
Is it technically possible to return a matplotlib object or something one can further manipulate inside a plot? 
Or maybe, just a util to extract a numpy array for a given observable... let's think about this

Reply:
We could return a plot object from the plot_observable, but if you would like to work with the numpy arrays directly there is already a function ( `get_obsv_from_json()` ) in python_utils.py that will extract those from the json output. Let me know if that's what you had in mind!

Reply:
yes I see.. it makes sense then to have this function as a quick tool to do plots, and then use the get_obs... to do more custom plots using raw numpy vectors